src/org/omegat/gui/editor/SegmentBuilder.java

```diff
@@ -290,7 +290,8 @@ private void createActiveSegmentElement(TMXEntry trans) throws BadLocationExcept
                 //translation exist
                 translationText = trans.translation;
             } else {
-                boolean insertSource = !Preferences.isPreference(Preferences.DONT_INSERT_SOURCE_TEXT);
+                boolean insertSource = Preferences.isPreferenceDefault(Preferences.DONT_INSERT_SOURCE_TEXT,
+                        Preferences.DONT_INSERT_SOURCE_TEXT_DEFAULT);
```

Comment:
It is not good to put feature default on Preferences, please put it in SegmentBuilder class as constant.